README.md

```diff
@@ -1,12 +1,32 @@
-# python-digitalocean
-
-This library provides easy access to Digital Ocean APIs to deploy droplets, images and more.
-
-[![](https://travis-ci.org/koalalorenzo/python-digitalocean.svg)](https://travis-ci.org/koalalorenzo/python-digitalocean)
-
-[![](https://img.shields.io/github/forks/badges/shields.svg?style=social&label=Fork)](https://travis-ci.org/koalalorenzo/python-digitalocean)
-[![](https://img.shields.io/github/stars/badges/shields.svg?style=social&label=Star)](https://travis-ci.org/koalalorenzo/python-digitalocean)
-[![](https://img.shields.io/github/watchers/badges/shields.svg?style=social&label=Watch)](https://travis-ci.org/koalalorenzo/python-digitalocean)
+<h1 align="center">python-digitalocean</h1>
```

Comment:
Please use Markdown.

Reply:
Unfortunately markdown does not support centering. I have to use HTML for this.